src/Plugin/ApplenewsComponentTypeBase.php

```diff
@@ -171,11 +168,17 @@ public function getComponentType() {
     return $this->pluginDefinition['component_type'];
   }
 
+  /**
+   *
```

Comment:
There are still a few like this one but this is already a huge improvement in coding standards so let's move one and merge it.